projects/js-packages/image-guide/src/MeasurableImage.ts

```diff
@@ -134,7 +134,7 @@ export class MeasurableImage {
 			const dimensions = await this.fetchFileDimensions( this.getURL() );
 			return dimensions.width < minSize || dimensions.height < minSize;
 		} catch ( err ) {
-			return false;
+			return true;
```

Comment:
This function has a comment on it that says what it returns when an image fails to load. If we're changing the behaviour we should update the comment too.

Reply:
Done in 99ca2a5.